### PR TITLE
Amended the openSUSE Channels table for openSUSE Leap Micro clients

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Fixed the old table listing openSUSE Channels - CLI, which was both incomplete and some rows were swapped.

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [x] master
- [ ] manager-4.3

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/22819
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Review by nodeg — COMMENTED** [review]
LGTM, however I cannot imagine how the rendered version does look like.

**Review by keichwa — CHANGES_REQUESTED** [review]
I guess there were 2 c&p errors.

**Comment by 0rnela** [comment]
> LGTM, however I cannot imagine how the rendered version does look like.
@nodeg 
4illustration

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc [hunk 1]
```diff
@@ -59,7 +59,7 @@ ifeval::[{uyuni-content} == true]
 .{opensuse} Channels - CLI
 |===
 
-| OS Version               | openSUSE Leap Micro 5.5            | openSUSE Leap Micro 5.4        | openSUSE Leap Micro 5.3
+| OS Version               | {leapmicro} 5.5                    | {leapmicro} 5.4                | {leapmicro} 5.3
 | Base Channel             | opensuse_micro5_5                  | opensuse_micro5_5              | opensuse_micro5_3
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:63]:
```suggestion
| Base Channel             | opensuse_micro5_5                  | opensuse_micro5_4              | opensuse_micro5_3
```

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ ifeval::[{uyuni-content} == true]
...
 | Client Channel           | opensuse_micro5_5-uyuni-client     | opensuse_micro5_4-uyuni-client | opensuse_micro5_4-uyuni-client
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:64]:
```suggestion
| Client Channel           | opensuse_micro5_5-uyuni-client     | opensuse_micro5_4-uyuni-client | opensuse_micro5_3-uyuni-client
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Maybe, sth went wrong.  Hope I got it right this time.

**Review by keichwa — APPROVED** [review]

**Pull request merged**